tests/AgDatabaseMove.Unit/StripedBackupSetTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Moq;
+using Xunit;
+
+namespace AgDatabaseMove.Unit
+{
+  public class StripedBackupSetTests
+  {
+
+    private static readonly StripedBackupEqualityComparer stripedBackupComparer = new StripedBackupEqualityComparer();
+    private static readonly BackupMetadataEqualityComparer backupComparer = new BackupMetadataEqualityComparer();
+
+    [Fact]
+    public void CombinesStripedBackups()
+    {
+      var backupChain = BackupOrder.GetBackupListWithStripes();
+
+      var stripedBackupSetChain = StripedBackupSet.GetStripedBackupSetChain(backupChain);
+
+      Assert.Equal(backupChain.Distinct(stripedBackupComparer).Count(), stripedBackupSetChain.Count());
+    }
+
+    [Fact]
+    public void DoesntCombineNonStripedBackups()
```

Comment:
Tests look good -- I see from your comments you tested on a local MSSQL server to verify it could restore from striped backups. 

Did you happen to verify that restoring without a striped backup chain works as intended as well?

Reply:
I think I did run that once, but not recently after the new changes
Will give it a go

Reply:
Just normal backups still work
and so does mixing and matching 👍🏽